addons/account/models/account_tax.py

```diff
@@ -213,12 +213,17 @@ def _constrains_name(self):
                     ])
             if duplicates := self.search(expression.OR(domains)):
                 raise ValidationError(
-                    _("Tax names must be unique!")
-                    + "\n" + "\n".join(_(
-                        "- %(name)s in %(company)s",
-                        name=duplicate.name,
-                        company=duplicate.company_id.name,
-                    ) for duplicate in duplicates)
+                    self.env._(
+                        "Tax names must be unique!\n%(taxes)s",
+                        taxes="\n".join(
+                            _(
```

Comment:
You should use `self.env._` here too because it is in a generator comprehension.